x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -246,9 +248,9 @@ export const CreateTrustedAppForm = memo<CreateTrustedAppFormProps>(
       return isGlobalEffectScope(trustedApp.effectScope);
     }, [trustedApp]);
 
-    const hideAssignmentSection = useMemo(() => {
-      return !isPlatinumPlus && (!isEditMode || (isGlobal && !isDirty));
-    }, [isEditMode, isGlobal, isDirty, isPlatinumPlus]);
+    const showAssignmentSection = useMemo(() => {
+      return isPlatinumPlus || (isEditMode && (!isGlobal || (wasByPolicy && isGlobal && isDirty)));
```

Comment:
Same as before, fixes a bug that displayed the assignment section after editing the form even if the exception was a global one

Reply:
no useMemo